packages/radix-ui-themes/package.json

```diff
@@ -37,7 +37,9 @@
     "dev:js:cjs": "tsc --project tsconfig-cjs.json --watch",
     "dev:js:esm": "tsc --project tsconfig-esm.json --watch",
     "dev:css": "postcss src/styles/index.css -o styles.css --watch",
-    "lint": "eslint \"src/**/*.ts*\"",
+    "lint": "yarn lint:js && yarn lint:css",
+    "lint:js": "eslint \"src/**/*.ts*\"",
+    "lint:css": "stylelint \"src/**/*.css\"",
```

Comment:
Are we enforcing linting at any point? My preference would be to require it before PRs can be merged

Reply:
Added a pre-publish script